parser_library/src/workspaces/workspace.cpp

```diff
@@ -319,6 +322,20 @@ bool workspace::load_and_process_config()
                 // else ignore, publish warning
             }
         }
+        if (asm_options_json.size() != 0)
+        {
+            std::map<std::string, std::string> asm_options;
+
+            if (asm_options_json.count("SYSPARM"))
+            {
+                asm_options.insert({ "SYSPARM", asm_options_json["SYSPARM"].get<std::string>() });
```

Comment:
will this code work when the user enters something different as string to the SYSPARM value?
e.g.
```
"asm_options" : {
  "SYSPARM" : 42
}
```

Reply:
I think this violates the schema, so I wouldn't see it as an issue.
On the other hand, we may want to impose the 255 characters limit imposed in the doc.

Reply:
It throws and then the exception is caught in `lsp::server`. It would be cleaner solution if we handled it directly in the configuration loading function